docs/source/plugins/developing_plugins.rst

```diff
@@ -2060,6 +2060,32 @@ subsequent sections.
             ctx.panel.set_state("event", "on_change_group_slice")
             ctx.panel.set_data("event_data", event)
 
+        def on_change_spaces(self, ctx):
```

Comment:
Thanks for updating the docs! One more place to document `ctx.spaces` is the [Execution context](https://docs.voxel51.com/plugins/developing_plugins.html#execution-context) section 😄